src/main/resources/application.yml

```diff
@@ -50,6 +50,24 @@ logging:
   level:
     com.dnastack.ga4gh.search.adapter.presto.PrestoHttpClient: DEBUG # logs queries when search fails
 
+auditing:
+  enabled: true
```

Comment:
I don't think we want to enable auditing by default when running search locally. Maybe this could be moved into an `audit` profile?